react-app/src/components/PortfolioScreen/PortfolioPanel.tsx

```diff
@@ -9,7 +9,53 @@ interface PortfolioPanelProps {
   portfolio: Portfolio;
 }
 
-const PortfolioPanel: React.FC<PortfolioPanelProps> = ({}) => {
+const ProfilePicture: React.FC<{ profile: Portfolio["profile"] }> = ({
+  profile,
+}) => {
+  const profilePicture = profile?.pictures?.profile;
+
+  return (
+    <div className={cn("flex", "justify-center", "mb-9")}>
+      {profilePicture ? (
+        <img
+          className={cn(
+            "rounded-full",
+            "w-[120px]",
+            "h-[120px]",
+            "sm:w-[180px]",
+            "sm:h-[180px]",
+            "sm:mr-9",
```

Comment:
I think if a margin is against a component that is not part of this component then i think it should be handled at the same level where both of these components are placed in, i.e should pass a className to the outer container, which applies for the 'mb-9' as well.